pio/src/iSpindel.cpp

```diff
@@ -472,6 +474,7 @@ bool uploadData(uint8_t service)
     sender.add("battery", Volt);
     sender.add("gravity", Gravity);
     sender.add("ID", ESP.getChipId());
+    sender.add("interval", my_sleeptime);
```

Comment:
I'm not sure about FHEM if it is additional data tolerant, I would rather not risk to break.